examples/classification_and_regression.ipynb

```diff
@@ -0,0 +1,811 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "9gmgN61SirPE"
+   },
+   "source": [
+    "# Gender Classification and Eyes Location Detection: A Two Task Problem"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "syR65Pohizmh"
+   },
+   "source": [
+    "In this example, we are going to implement a multi-task problem. We try to identify the gender of the people, as well as locating their eyes in the image. Hence, we have two different tasks: classification (to identify the gender) and regression (to find the location of the eyes). We are going to use a single network (a CNN) to perform both tasks, however, we will need to apply different loss functions, each proper to a specific task. For this example we will need to install a few libraries (such as OpenCV, wget). If you don't have them, they can be installed as below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "VAG5NDFCiqxG",
+    "outputId": "bb6df7c7-0197-42aa-a8be-f95737254b57"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Looking in links: https://download.pytorch.org/whl/nightly/cu102/torch_nightly.html\n",
+      "Requirement already satisfied: torchvision in /usr/local/lib/python3.7/dist-packages (0.12.0.dev20211124+cu102)\n",
+      "Requirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from torchvision) (1.19.5)\n",
+      "Requirement already satisfied: pillow!=8.3.*,>=5.3.0 in /usr/local/lib/python3.7/dist-packages (from torchvision) (7.1.2)\n",
+      "Requirement already satisfied: torch==1.11.0.dev20211124+cu102 in /usr/local/lib/python3.7/dist-packages (from torchvision) (1.11.0.dev20211124+cu102)\n",
+      "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.7/dist-packages (from torch==1.11.0.dev20211124+cu102->torchvision) (3.10.0.2)\n",
+      "Requirement already satisfied: poutyne in /usr/local/lib/python3.7/dist-packages (1.7)\n",
+      "Requirement already satisfied: torch in /usr/local/lib/python3.7/dist-packages (from poutyne) (1.11.0.dev20211124+cu102)\n",
+      "Requirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from poutyne) (1.19.5)\n",
+      "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.7/dist-packages (from torch->poutyne) (3.10.0.2)\n",
+      "Requirement already satisfied: wget in /usr/local/lib/python3.7/dist-packages (3.2)\n",
+      "Requirement already satisfied: opencv-python in /usr/local/lib/python3.7/dist-packages (4.1.2.30)\n",
+      "Requirement already satisfied: numpy>=1.14.5 in /usr/local/lib/python3.7/dist-packages (from opencv-python) (1.19.5)\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install -U --pre torchvision -f https://download.pytorch.org/whl/nightly/cu102/torch_nightly.html\n",
```

Comment:
```suggestion
```
Since the dataset is fixed in the last torchvision version, I don't think we need to install the nightly version anymore.